website/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 1
 
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
-| [ExomeGermlineSingleSample_v2.4.3](https://github.com/broadinstitute/warp/releases) | December 07, 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in dsde-pipelines or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+| [ExomeGermlineSingleSample_v3.0.0](https://github.com/broadinstitute/warp/releases) | November, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in dsde-pipelines or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
```

Comment:
Can we update this link to go to the search page? If you include `ExomeGermlineSingleSample` as the search term it should work

Reply:
If you add a full version number, it returns the whole list of releases. I added that.